signac/contrib/collection.py

```diff
@@ -55,8 +55,8 @@
     "null": type(None),
 }
 
-
-MAX_DEFAULT_ID = int("F" * 32, 16)
+MAX_DEFAULT_ID_LENGTH = 32
```

Comment:
Shouldn't this be `DEFAULT_ID_LENGTH` or simply `ID_LENGTH`?

Reply:
I changed it to `ID_LENGTH`.